File: output_number_1/raw_outputs/qwen_coder/file_004_naive.js
const { exec } = require('child_process');

function resizeImage(imagePath) {
  const newImagePath = imagePath.replace('.jpg', '_resized.jpg');
  exec(`convert ${imagePath} -resize 50% ${newImagePath}`, (error, stdout, stderr) => {
    if (error) {
      console.error(`Error resizing image: ${stderr}`);
      return;
    }
    console.log(`Image resized successfully to ${newImagePath}`);
  });
}

module.exports = resizeImage;

File: latest_ran_output/qwen_coder/file_004_naive.py
import subprocess

def resize_image(image_path, output_path, size):
    try:
        subprocess.run(['convert', image_path, '-resize', size, output_path], check=True)
        return True
    except subprocess.CalledProcessError as e:
        print(f"Error resizing image: {e}")
        return False